### PR TITLE
update readme.md, s/packageJS/fastOptJS/

### PR DESCRIPTION
**Pull request by vendethiel**
since that was renamed in scalaJS

nice plugin btw :+1:

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ This will attempt to perform an update without refreshing the page every time `f
 
 Nonetheless, for the bulk of javascript libraries these limitations are acceptable. As long as you're not doing anything too crazy, `updateBrowsers` but should suffice for most applications.
 
-You can force the clean-up-and-reboot to happen from the browser via the shortcut Ctrl-Alt-Shift-Enter if you simply wish to reset the browser to a clean state.
+You can force the clean-up-and-reboot to happen from the browser via the shortcut Ctrl-Alt-Shift-Enter if you simply wish to reset the browser to a clean state.pa
```
Comment (sjrd) [inline readme.md]: I guess this should not be there.

Comment (vendethiel) [inline readme.md]: Whoops, my bad. fixed.

**Comment by vendethiel** [comment]
@lihaoyi ?

**Comment by jodersky** [comment]
thanks, this was fixed in #33

**Pull request closed without merging**